src/Backend/Modules/Slideshows/Layout/Templates/Edit.html.twig

```diff
@@ -20,6 +20,15 @@
         </label>
         {% form_field title %} {% form_field_error title %}
       </div>
+
```

Comment:
you should only show/add this dropdown if there are more than 1 possible template, just like we do in the Content Blocks module